### ISSUE TITLE
Missing start script?

### ISSUE THREAD

**Issue by DanielRuf** [diverent2/flagfinder#3]
Hi, it seems the `__sapper__` directory is missing. Not sure if there should be one but using the `start` script does not work.

```shell
yarn start
yarn run v1.19.0
warning package.json: No license field
$ node __sapper__/build
module.js:550
    throw err;
    ^

Error: Cannot find module '/Users/danielruf/projects/flagfinder/__sapper__/build'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by diverent2** [comment]
I've finally managed to update the readme.  
This _should_ clarify things.
**TLDR:** You will need to build the app first before you can run it:

```
yarn build && yarn start
```

If you still having issues, let me know :)

**Comment by DanielRuf** [comment]
Awesome, thank you very ,uch =)
